torch/jit/_freeze.py

```diff
@@ -140,4 +144,12 @@ def optimize_frozen_module(mod):
         assert "batch_norm" not in str(frozen_mod.graph)
 
     """
-    torch._C._jit_pass_optimize_frozen_graph(mod.graph)
+    # xxx: keep in sync with frozen_graph_optimization.cpp
+    # intentionally duplicated to make to make it easier to create custom optimization sequence
+    torch._C._jit_pass_remove_dropout(mod._c)
+    if optimize_numerics:
+        # run a couple times to capture Conv -> Mul -> Add etc
+        for _ in range(2):
```

Comment:
it's not very scientific, you probably want to iterate till the fixed point :) but it's unrelated to this diff

Reply:
yea, that's kind of a todo, i'm not convinced it really matters but it would be a good follow up